Error: Cannot find module '/Users/me/helia-examples/examples/helia-101/index.js'

Steps to reproduce on macOS with node v20.5.0:

~/
gh repo clone ipfs-examples/helia-examples
cd helia-examples/examples/helia-101
npm install
npm start
=> 
```
> helia-101@1.0.0 start
> node index.js

node:internal/modules/cjs/loader:1051
  throw err;
  ^

Error: Cannot find module '/Users/kopeboy/helia-examples/examples/helia-101/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.5.0
npm ERR! Lifecycle script `start` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: helia-101@1.0.0
npm ERR!   at location: /Users/kopeboy/helia-examples/examples/helia-101
```

--- a/examples/helia-101/README.md
+++ b/examples/helia-101/README.md
@@ -60,7 +60,7 @@ Make sure you have installed all of the following prerequisites on your developm
 
 ```console
 > npm install
-> npm start
+> node ./101-basics.js
 ```
 
 ## Usage

--- a/examples/helia-101/package.json
+++ b/examples/helia-101/package.json
@@ -7,7 +7,7 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "node 101-basics.js",
     "serve": "npm run start",
     "test": "test-node-example test/*"
   },

